src/components/Icon/Icons/Bandcamp.jsx

```diff
@@ -8,11 +8,18 @@ import createIconComponent from '../utils/createIconComponent';
 const BandcampIcon = createIconComponent({
```

Comment:
@gmzjuliana I haven't generated icons in a long time. I executed the `yarn gen:icons` command and that seemed to update a few icons

Reply:
It changed the code, but the icons looks the same 😄